modules/js/perf/perf_imgproc/perf_cvtcolor.js

```diff
@@ -4,20 +4,36 @@ if (isNodeJs) {
   var Benchmark = require('benchmark');
   var cv = require('../../opencv');
   var HelpFunc = require('../perf_helpfunc');
-  var Base = require('../base');
 } else {
   var paramsElement = document.getElementById('params');
   var runButton = document.getElementById('runButton');
   var logElement = document.getElementById('log');
 }
 
-cv.onRuntimeInitialized = () => {
+var cvSize
+
+async function main() {
```

Comment:
The cv.onRuntimeInitialized path is still needed if cv is not a promise. So I suppose the code logic should be:
```js
if (cv instanceof Promise) {
  cv = await cv;
  main();
} else {
  cv.onRuntimeInitialized = main;
}
```